src/components/EditionModal/EditionModal.jsx

```diff
@@ -88,20 +94,19 @@ const InfoSlide = ({
         const fieldLabel = fieldLabels[fieldName]
         const chooserProps = resultWithArgs(fieldSpec, 'chooserProps', [doc])
         const sectionProps = resultWithArgs(fieldSpec, 'sectionProps', [doc])
+        const hasSelector = TYPES_WITH_SELECTOR[fieldSpec.type]
         return (
           <FieldSection
             key={fieldName}
             label={fieldLabel}
             value={fieldSpec.getValue(doc)}
-            onClick={
-              fieldSpec.immediate ? null : () => onRequestChooseField(fieldName)
-            }
+            onClick={hasSelector ? () => onRequestChooseField(fieldName) : null}
             onChange={
-              fieldSpec.immediate
-                ? value => {
+              hasSelector
+                ? null
```

Comment:
Don't we want to pass `undefined`, to act as if the prop wasn't passed at all? (for example if there is a default prop)